packages/usa-tooltip/src/index.js

```diff
@@ -373,31 +373,22 @@ const setUpAttributes = (tooltipTrigger) => {
   // place the text in the tooltip
   tooltipBody.textContent = tooltipContent;
 
-  return { tooltipBody, position, tooltipContent, wrapper };
-};
-
-/**
- * Set up tooltip trigger event handlers.
- * @param {HTMLElement} tooltipTrigger The element that creates the tooltip
- */
-const setUpEvents = (tooltipTrigger) => {
   tooltipTrigger.addEventListener("mouseover", onTooltipActivate);
   tooltipTrigger.addEventListener("focusin", onTooltipActivate);
   tooltipTrigger.addEventListener("mouseout", onTooltipDeactivate);
   tooltipTrigger.addEventListener("focusout", onTooltipDeactivate);
+
+  return { tooltipBody, position, tooltipContent, wrapper };
 };
 
 // Setup our function to run on various events
 const tooltip = behavior(
   {},
   {
     init(root) {
-      selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
-        setUpAttributes(tooltipTrigger);
-        setUpEvents(tooltipTrigger);
-      });
+      selectOrMatches(TOOLTIP, root).forEach(setup);
     },
-    setup: setUpAttributes,
+    setup,
```

Comment:
## Setup pattern

It appears that tooltip is the only component that follows the pattern of renaming and exporting a `setup` pattern. 

Initially, I was just referring to exporting both `setUpAttributes` and `setUpEvents` as methods.

```diff
const tooltip = behavior(
  {},
  {
    init(root) {
      selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
        setUpAttributes(tooltipTrigger);
        setUpEvents(tooltipTrigger);
      });
    },
+   setUpAttributes,
+   setUpEvents
    getTooltipElements,
    show: showToolTip,
    hide: hideToolTip,
  }
);
```

We could instead follow the pattern set by Modal:

It uses a `setUpModal` function to handle all setup tasks

https://github.com/uswds/uswds/blob/3175c0258a63e6e464ea8284bb90f452f46463a1/packages/usa-modal/src/index.js#L286-L304

Then it calls that as part of its `init`.

We could follow this pattern by maintaining each function and calling them in one `setupTooltip` function.

```js
const setUpAttributes = (tooltipTrigger) => {
   ...
}

const setUpEvents = (tooltipTrigger) => {
  ...
}

const setupTooltip = (tooltipTrigger) => {
  setUpAttributes(tooltipTrigger);
  setUpEvents(tooltipTrigger);
}
```

What do you think @amyleadem @mejiaj ? Would it be a regression to remove the `setup` method? If so, we could export it as `setup: setUpTooltip`

Reply:
@mahoneycm this is a public method, so it would be a breaking change if users rely on it and we remove it.

Like you mentioned, there are alternatives like:
1. Keeping the method, but calling a different function.
2. Providing an alternative method and guidance for migrating.

I do agree this might not follow a conventional pattern, but that seems out-of-scope for the work outlined in the problem statement. If you agree can you create a separate issue to address this?